tests/unit/query/createFilter.ts

```diff
@@ -267,7 +267,7 @@ registerSuite({
 				value: 5,
 				path: createJsonPointer('key', 'key2')
 			};
-			const pickFirstItem = [
+			const pickFirstItem: FilterArray<NestedObj> = [
```

Comment:
They should spell out Object, as the style guide makes it clear that we shouldn't be unnecessarily shortening variable names. I've updated them both.